frontend/tests/features/wheresMyGene/shareLink.test.ts

```diff
@@ -44,10 +44,13 @@ const SEXES = [
 ];
 const COMPARE = "disease";
 
+const CELL_TYPES = ["CD4-positive helper T cell"];
+
 const SHARE_LINK =
   `${TEST_URL}/gene-expression?` +
   // (thuang): Tissue params include a tissue id and a tissue name to test that we support both
-  "compare=disease&datasets=d8da613f-e681-4c69-b463-e94f5e66847f%2Cde2c780c-1747-40bd-9ccf-9588ec186cee&diseases=MONDO%3A0100096&ethnicities=unknown&sexes=PATO%3A0000383%2CPATO%3A0000384&tissues=blood%2CUBERON%3A0002048&genes=DPM1%2CTNMD%2CTSPAN6&ver=2";
+  // (seve): cellType params includes a incorrect cellType to test that we filter out invalid params
```

Comment:
nit: I think there are two invalid cell types? DOG and CAT. If so, can we explicitly list them out in the comment too, so people don't have to guess? (Especially in this case the comment says one incorrect cellType instead of two. Unless DOG or CAT is actually a valid cell type and I didn't know 😆 )

Reply:
^ @seve PTAL this thank you!